trigger.sh

```diff
@@ -27,6 +27,13 @@ for BRANCH in master R5.0; do
     cd $mydir
     if [ "$result" -ne 0 ]; then
        echo "$(date) Found build number for ${BRANCH}: ${BUILD_NUMBER}, missing in the registry. Starting build." | tee -a "$log_file"
```

Comment:
Please keep this `echo` line just above `zuul enqueue-ref` line, otherwise it becomes misleading.